CanalTP/Chaos
# Clean line section with migrate file

# Description

This PR delete some outdated line_section data

## Issue

Issue link: BOT-1915

## How to test

Here are the following steps to test this pull request:

With `source venv/bin/activate`
- Use the command `CHAOS_CONFIG_FILE=default_settings.py ./manage.py db upgrade` to launch the migration
- Use the command `CHAOS_CONFIG_FILE=default_settings.py ./manage.py db downgrade` to launch the donwgrade of the migration
- Check in bdd and do some API call.
